rpmlint/checks/FileDigestCheck.py

```diff
@@ -8,24 +8,28 @@
 class FileDigestCheck(AbstractCheck):
     def __init__(self, config, output):
         super().__init__(config, output)
-        self.digest_groups = {}
+        self.digest_configurations = {}
         self.follow_symlinks_in_group = {}
         for group, values in self.config.configuration['FileDigestLocation'].items():
-            self.digest_groups[group] = [Path(p) for p in values['Locations']]
```

Comment:
Could you also rename the `group` for loop variable? Also in some other for loops that deal with `digests_configurations`.